CMakeLists.txt

```diff
@@ -23,6 +23,12 @@ endif()
 
 #===========================================================================================
 # Strip garbage
+if (WIN32)
```

Comment:
This is problem of MSVC exclusive and this doesn't happen on MinGW builds at all. So, you can freely use the `MSVC` variable here.